packages/inquirer/package.json

```diff
@@ -60,15 +60,19 @@
   "license": "MIT",
   "dependencies": {
     "@inquirer/figures": "^1.0.3",
-    "@ljharb/through": "^2.3.13",
```

Comment:
removing just this one should already have a significant impact, it does bring quite a few dependencies recursively!